pkg/envs/bool.go

```diff
@@ -0,0 +1,14 @@
+package envs
+
+import (
+	"os"
+	"strconv"
+)
+
+func IsTrue(name string) (is bool) {
+	if val, ok := os.LookupEnv(name); ok {
+		is, _ = strconv.ParseBool(val)
```

Comment:
I prefer to log all errors. Never know what issue you may face

Reply:
logged to testkubes default logger